web/app/js/components/Tap.jsx

```diff
@@ -83,15 +104,15 @@ class Tap extends React.Component {
     if (!e.wasClean) {
       this.setState({
         error: {
-          error: `Websocket [${e.code}] ${e.reason}`
+          error: `Websocket close error: [${e.code}] ${e.reason ? e.reason : wsCloseCodes[e.code]}`
```

Comment:
Since the websocket codes are less known that http status codes, it might be worth displaying the text all the time here, like
```
error: `Websocket close error [${e.code}: ${wsCloseCodes[e.code]}] ${e.reason ? ":" : ""} ${e.reason}`
```

Reply:
![screen shot 2018-09-12 at 10 09 31 am](https://user-images.githubusercontent.com/236915/45441754-cad1aa80-b674-11e8-9033-f90785feeb5e.png)